pkgs/by-name/ar/argc/package.nix

```
@@ -37,6 +38,11 @@ rustPlatform.buildRustPackage rec {
 
   disallowedReferences = lib.optional (!canExecuteHost) buildPackages.argc;
 
+  env = {
+    LANG = "en_US.UTF-8";
```

I found that `LANG=C.UTF-8` would also work, which sounds more language- and location-neutral.

```suggestion
    LANG = "C.UTF-8";
```